src/python/gudhi/persistence_graphical_tools.py

```diff
@@ -20,6 +20,20 @@
 __copyright__ = "Copyright (C) 2016 Inria"
 __license__ = "MIT"
 
+@lru_cache(maxsize=1)
```

Comment:
Is this still useful in this version?

Reply:
You are right, `import matplotlib` will be catched before this call. I remove it.

Reply:
Maybe you can add it back with the more recent change?